.classpath

```diff
@@ -45,14 +45,6 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" path="target/generated-sources/annotations">
```

Comment:
Is this deletion in the .classpath file intentional?  I don't know what it does.

Reply:
Not intentional, I had believed I had removed from commit, but apparently not.
Eclipse just loves to mess with class path if thinks it doesn't agree with the pom.xml.
Sorry.